dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/PageParams.java

```diff
@@ -43,6 +43,10 @@ public class PageParams {
   private static final int DEFAULT_PAGE = 1;
   private static final int DEFAULT_PAGE_SIZE = 50;
 
+  public static PageParams single() {
```

Comment:
Just throwing this out there. I have see  "first()" used in libs, that could be a potential name here as well. No strong opinions